src/style.css

```diff
@@ -0,0 +1,93 @@
+html {
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 100px;
+}
+
+.block {
+  display: flex;
+  line-height: 300px;
+  justify-content: center;
+  height: 300px;
+  min-height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
```

Comment:
why use `min-height` here?

you can achieve the same without `flex-grow` if you do not hardcode `flex-basis` in pixels. think about other units